spec/factories/authentication_tokens.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :authentication_token do
+    token { "MyString" }
+    user { nil }
+    expires_at { "2022-11-04 10:02:04" }
```

Comment:
I see, I assume that this factory was auto-generated or something, as you would use DateTime.now + x.hours here or something similar.